pcsx2/IopBios.cpp

```diff
@@ -102,6 +102,12 @@ void Hle_SetElfPath(const char* elfFileName)
 	Console.WriteLn("HLE Host: Set 'host:' root path to: %s\n", hostRoot.c_str());
 }
 
+void Hle_SetIsoPath(const char* isoFilename)
```

Comment:
Don't really see why this needs an extra function which does the exact same thing as the one above it. Renaming it to "SetHostRoot" or something would be better.